matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/Event.java

```diff
@@ -687,6 +690,59 @@ public boolean isSent() {
         return urls;
     }
 
+    /**
+     * @return all the encrypted file infos defined in the event.
+     */
+    public List<EncryptedFileInfo> getEncryptedFileInfos() {
+        ArrayList<EncryptedFileInfo> urls = new ArrayList<>();
```

Comment:
Bad copy/paste: rename `urls` to sth like `encryptedFileInfos`
Also the type should be `List` and not `ArrayList`